apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Tests/MockBigtableClient.cs

```diff
@@ -45,9 +31,9 @@ public class MockBigtableClient : BigtableClient
         public override Task<MutateRowResponse> MutateRowAsync(MutateRowRequest request, CallSettings callSettings = null) =>
             throw new RequestMadeException();
         public override MutateRowsResponse MutateRows(MutateRowsRequest request, CallSettings callSettings = null) =>
```

Comment:
If you don't override these methods at all, you'll get a NotImplementedException, won't you? Can't you just rely on that for anything you don't want to handle?

Reply:
Yes, I believe so, but I believe when I added this, I wanted to make sure we weren't seeing NotImplementedException for some other reason that wasn't due to the request being made. However, I guess that's pretty unlikely, so maybe it's safe to rip this out.